service_management/azure/test/integration/blob/create_page_blob_test.rb

```diff
@@ -80,4 +80,28 @@
       end
     end
   end
+
+  describe '#create block blob' do
+    let(:container_name) { 'testcontainer' }
+
+    before do
+      subject.create_container container_name
+    end
+
+    after do
+      subject.delete_container container_name
+    end
+
+    it 'should create a block blob with spaces in name' do
+      blob_name = 'blob with spaces'
+      blob = subject.create_block_blob container_name, blob_name, 'content'
+      blob.name.must_equal blob_name
+    end
+
+    it 'should create block blob with complex in name' do
```

Comment:
is this validating something additional to "should create a block blob with spaces in name" and "creates page blob with non uri encoded path"?